ts/Extensions/DashboardPlugins/HighchartsSyncHandlers.ts

```diff
@@ -94,7 +94,7 @@ const configs: {
         tooltipEmitter: [
             'tooltipEmitter',
             function (this: ComponentTypes): Function | void {
-                if (this instanceof HighchartsComponent) {
+                if (this instanceof (HighchartsComponent || (window as any).HighchartsComponent)) {
```

Comment:
Define the component on window to prevent `any`:
```
/* *
 *
 *  Declarations
 *
 * */

declare global {
  interface Window {
    HighchartsComponent?: typeof HighchartsComponent;
  }
}
```